src/Layout.elm

```diff
@@ -81,7 +81,7 @@ header currentPath =
                         (Element.text "Code Reading Club")
                 }
             , Element.column [ Element.paddingXY 0 8, Element.spacing 8 ]
-                [ highlightableLink currentPath Pages.pages.blog.directory "Code Reading Countdown"
+                [ highlightableLink currentPath Pages.pages.blog.directory "Blog"
```

Comment:
We used to all these 'adaventures' but never posted any... so maybe we remove the link altogether until we have meaningful content there?

Reply:
@Nikomus  Shall I delete and release? What do you tihnk?

Reply:
I'm just gonna go ahead. We can put it back if we change our minds. Better to get those other updates published.